tests/test_eload_ingestion.py

```diff
@@ -35,13 +36,32 @@ def tearDown(self):
             os.remove(ingest_csv)
         self.eload.eload_cfg.content = self.original_cfg
 
-    def _mock_mongodb_client(self):
-        m_db = mock.Mock()
-        m_db.list_database_names = mock.Mock(return_value=[
-            'eva_ecaballus_30',
-            'eva_hsapiens_grch38'
-        ])
-        return m_db
+    # def _mock_mongodb_client(self):
+    #     m_db = mock.Mock()
+    #     m_db.list_database_names = mock.Mock(return_value=[
+    #         'eva_ecaballus_30',
+    #         'eva_hsapiens_grch38'
+    #     ])
+    #     return m_db
+
+    def _patch_get_dbname(self, db_name):
+        m_get_db_name = patch(
+            'eva_submission.eload_ingestion.get_new_variant_warehouse_db_name_from_assembly_and_taxonomy',
+            autospec=True,
+            return_value=db_name
+        )
+        return m_get_db_name
+
+    def _patch_metadata_handle(self):
```

Comment:
I think this can be used in a few more of the tests (probably all of them actually).

Reply:
Added